go/private/actions/archive.bzl

```diff
@@ -212,5 +212,4 @@ def emit_archive(go, source = None, _recompile_suffix = "", recompile_internal_d
         cgo_deps = depset(transitive = [cgo_deps] + [a.cgo_deps for a in direct]),
         cgo_exports = cgo_exports,
         runfiles = runfiles,
-        mode = go.mode,
```

Comment:
Please also update the providers.rst file

Reply:
thanks, done